### PR TITLE
Fix error in `resetPosition`

### PR DESCRIPTION
**Pull request by fjakobs**
Fixes https://github.com/webdriverio-community/wdio-vscode-service/issues/70

### CODE REVIEW

File: src/pageobjects/workbench/Input.ts [hunk 1]
```diff
@@ -244,7 +244,7 @@ export abstract class Input extends BasePage<AllInputLocators> {
     private async resetPosition (): Promise<void> {
         const text = await this.getText()
         await this.clear()
-        await this.setText(text)
+        await this.setText(text || "")
```
Comment (christian-bromann) [inline src/pageobjects/workbench/Input.ts:247]:
```suggestion
        await this.setText(text || '')
```

**Review by christian-bromann — APPROVED** [review]
Thanks, LGTM 👍

**Comment by fjakobs** [comment]
Thanks for approving the change. The tests are failing, but I can't tell if it's a regression introduced by my change or just flakiness.

**Comment by christian-bromann** [comment]
@fjakobs let me take a look at this later today

**Comment by fjakobs** [comment]
popping this up your stack

**Comment by christian-bromann** [comment]
Thanks for the fix, I will update the package shortly.

**Comment by christian-bromann** [comment]
Congratulations on your first WebdriverIO contribution! This project can't live without the participation of the community. We would love to see more from you, so let us know if we can help to find interesting areas for you to contribute to. We appreciate you 🙏 ❤️

**Comment by fjakobs** [comment]
Thanks @christian-bromann

**Pull request merged**